src/resolver/data_url.zig

```diff
@@ -71,22 +71,28 @@ pub const PercentEncoding = struct {
             }
         }
 
-        if (ret) |some| return allocator.shrink(some, ret_index);
+        if (ret) |some| return some[0..ret_index];
         return null;
     }
 };
 
 pub const DataURL = struct {
+    url: bun.String = bun.String.empty,
     mime_type: string,
     data: string,
     is_base64: bool = false,
 
-    pub fn parse(url: string) ?DataURL {
+    pub fn parse(url: string) !?DataURL {
         if (!strings.startsWith(url, "data:")) {
             return null;
         }
 
-        const comma = strings.indexOfChar(url, ',') orelse return null;
+        var result = try parseWithoutCheck(url);
```

Comment:
return try parseWithoutCheck(url);

Reply:
but it doesn't matter

Reply:
it's actually compiler error, not sure if it's a zig bug